backend/app/views/spree/admin/shared/_sidebar_left.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='sidebar'>
```

Comment:
Single quoted attributes :cry:

But more importantly: the partial is called `_sidebar_left` but the class name is `sidebar` and the children use `.sidebar-left-header`.